recipe.cake

```diff
@@ -291,11 +291,11 @@ Task("Prepare-NuGet-Packages")
     .Does(() =>
 {
     // Copy legal documents
-    CleanDirectory(BuildParameters.Paths.Directories.NuGetNuspecDirectory + "/lib");
+    CleanDirectory(BuildParameters.Paths.Directories.NuGetNuspecDirectory + "/lib/net48");
```

Comment:
Any reason why you changed this to only clean the .NET 4.8 directory?

Shouldn't this clean out everything?

Reply:
There is no real reason, no. The CleanDirectory here is/was really being used to ensure that the directory existed, since the nuspec directory is cleaned out elsewhere during the build.

I have no issue with switching to the CleanDIrectories alias, and doing both lib and lib/net48.

Assuming that this is what you are suggesting.

Reply:
I have went with a combination of the two approaches here. Let's ensure that the `lib` directory is cleaned, and then ensure that the `net48` folder exists.